src/[% if repository_provider == 'github.com' %].github[% endif %]/workflows/build.yml.jinja

```
@@ -38,3 +38,5 @@ jobs:
         overwrite: true
         file_glob: true
     - uses: pypa/gh-action-pypi-publish@v1.9.0
```

don't forget to bump, this release includes the bug fix
```suggestion
    - uses: pypa/gh-action-pypi-publish@v1.10.1
```